cps/CP95/CP95.md

```diff
@@ -181,4 +181,4 @@ TBD - this will be determined with assistance from the Centrifuge Credit Group.
 
 Link to the RFC (POP) on the Forum: https://gov.centrifuge.io/t/pop-new-silver-ns3/5603
 
-Link to the onchain vote (democracy proposal): https://centrifuge.subsquare.io/democracy/proposals/5
+Link to the onchain vote (democracy proposal): https://centrifuge.subsquare.io/council/motions/86

```

Comment:
Should probably be:

Link to the onchain vote (council motion + referendum): https://centrifuge.subsquare.io/council/motions/86